csp_billing_adapter/csp_cache.py

```diff
@@ -110,25 +110,25 @@ def add_usage_record(record: dict, cache: dict) -> None:
 
 def cache_meter_record(
     cache: dict,
-    record_id: str,
+    record_ids: list,
     dimensions: dict,
     metering_time: str
 ) -> None:
     """
     Update the cache data store to reflect the fact that a successful CSP
-    metering operation was performed, storing the record_id returned by
+    metering operation was performed, storing the record_ids returned by
     the CSP, the billing dimensions submitted in the metering operation,
     the time at which it was performed, and the time after which the next
     billing operation will be performed.
 
-    :param record_id:
+    :param record_ids:
         The record id returned by the CSP to track the billing
         submission.
     :param dimensions: The billing dimensions submitted to the CSP.
     :param metering_time: The time at which the submission occurred.
     """
     cache['last_bill'] = {
         'dimensions': dimensions,
-        'record_id': record_id,
+        'record_ids': record_ids,
```

Comment:
This name change represents a modification of the on-disk data structure in the cache. We would need to ensure that any code that references this field appropriately handles this name change...

Reply:
@smarlowucf will changing the name of this field in the cache data structure cause issues?